src/main/java/com/artipie/front/ui/repository/RepoList.java

```diff
@@ -47,4 +63,76 @@ public RepoList(final RepositoryService repository, final SettingsService settin
             )
         );
     }
+
+    /**
+     * Repository information.
+     * @since 0.1.3
+     */
+    public static class Repo {
+        /**
+         * Repository name.
+         */
+        @SuppressWarnings("PMD.AvoidFieldNameMatchingMethodName")
```

Comment:
hm, one suppression on class level should be enough

Reply:
@andpopov what about this one? Is qulice still complaining if you move `@SuppressWarnings("PMD.AvoidFieldNameMatchingMethodName")` to `Repo` class declaration and remove suppressions under the fields?

Reply:
qulice works correctly, fixed.
Thanks